src/bullets/data_source/data_source_fmp.py

```diff
@@ -189,6 +220,7 @@ def _get_cached_cash_flow_statement(self, symbol: str, wanted_date: date):
 
     def _store_price_points(self, symbol: str, start_date: datetime.date, end_date: datetime.date = None,
                             limit: int = 1000):
+        print("I am called with " + str(start_date) + " to " + str(end_date))
```

Comment:
I think you forgot to remove this line after debugging

Reply:
Indeed, thanks!

Reply:
Fixed